src/namer.h

```diff
@@ -139,6 +139,10 @@ class Namer {
     return Format(s, config_.variables);
   }
 
+  virtual std::string Variable(const ::reflection::Field &f) const {
```

Comment:
hi, sorry for the pedantic back and forths. This should go in `bfbs_namer.h`.

Context: Since we're migrating from the IdlParser representations to the reflection representations, but want to
use `Namer` for both, we have `IdlNamer` and `BfbsNamer` to offer typed overloads where `Namer` itself is string-ly typed.